src/server/server_family.cc

```diff
@@ -1429,7 +1429,7 @@ void ServerFamily::Info(CmdArgList args, ConnectionContext* cntx) {
   if (should_enter("SERVER")) {
     auto kind = ProactorBase::me()->GetKind();
     const char* multiplex_api = (kind == ProactorBase::IOURING) ? "iouring" : "epoll";
-
+    append("thread_count", service_.proactor_pool().size());
```

Comment:
That's good but please move it after tcp_port, thanks 🙂

Reply:
@romange sir i have done that could you please take a look on it.